ngraph/test/models/onnx/model_editor/reference/subgraph__initializer_without_matching_input_tail_cut.prototxt

```diff
@@ -0,0 +1,120 @@
+ir_version: 3
+producer_name: "tomdol"
```

Comment:
@tomdol Why do we need to have this field?

Reply:
We don't, it's optional. I can remove this field but it will require a full run of CI.